### ISSUE TITLE
Unable to get property 'ADDITION' of undefined or null reference

### ISSUE THREAD

**Issue by poloka** [cerner/xfc#3]
Please fill out the below template as best you can.
--------------------------------------------------------

### Description of Issue
Receiving the following error when embedded within an iframe
```
Unable to get property 'ADDITION' of undefined or null reference
```

#### Project Version
xfc-1.0.1

#### Additional Details (optional)
IE 11.0.32 build 11.0.9600.18349CO ([KB3160005](https://support.microsoft.com/en-us/help/3160005/ms16-063-security-update-for-internet-explorer-june-14,-2016))
Also logged [issue](https://github.com/webmodules/mutation-observer/issues/7) against the mutation-observer for additional assistance.
The line of code that is throwing the error was [here](https://github.com/webmodules/mutation-observer/blob/78851e5da1a4b06c83883959e8557fcae05ca4d5/index.js#L491).  

### Steps to Reproduce the Issue
Unable to create in sample application with local, only reproducible on a citrix machine.

### Expected Outcomes
Content loads without javascript errors.

**Comment by mhemesath** [comment]
XFC never attempts to get a property called `ADDITION`; the property name doesn't even exist in the repo. What is leading you to believe this is an XFC issue?

**Comment by poloka** [comment]
The error received is referenced within the dependency noted in xfc's [package.json](https://github.com/cerner/xfc/blob/master/package.json#L24).  Not directly an issue with XFC but rather in the dependency being utilized; hence why I've also logged an issue with the mutation-observer in attempting to identify where the problem is originating from.

**Comment by poloka** [comment]
Created a [PR](https://github.com/webmodules/mutation-observer/pull/8) for the mutation-observer in case the MutationEvent has been deprecated in its usage.  Looks like the mutation-observer project hasn't been updated in over 2 years so concerned that this change will not be accepted.  Are they any other possibilities for polyfills for usage other than this project?

**Comment by kafkahw** [comment]
Fixed in PR #4
